src/java.base/share/classes/java/lang/reflect/Executable.java

```diff
@@ -280,18 +282,23 @@ public Set<AccessFlag> accessFlags() {
      * underlying executable takes no parameters.  Note that the
      * constructors of some inner classes may have an implicitly
      * declared parameter in addition to explicitly declared ones.
-     * Also note that as a <a
-     * href="{@docRoot}/java.base/java/lang/reflect/package-summary.html#LanguageJvmModel">modeling
+     * Also note that compact constructors of a record class may have
+     * implicitly ({@linkplain java.compiler/javax.lang.model.util.Elements.Origin#MANDATED mandated}) declared parameters.
```

Comment:
I think a better version of this edit is

`{@linkplain java.compiler/javax.lang.model.util.Elements.Origin#MANDATED implicitly declared}`